cmd/osv-scanner/main.go

```diff
@@ -68,21 +71,22 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Usage:   "sets the output format",
 				Value:   "table",
 				Action: func(context *cli.Context, s string) error {
-					switch s {
-					case
-						"table",
-						"json", //nolint:goconst
-						"markdown":
+					if slices.Contains(reporter.Format(), s) {
 						return nil
 					}
 
-					return fmt.Errorf("unsupported output format \"%s\" - must be one of: \"table\", \"json\", \"markdown\"", s)
+					return fmt.Errorf("unsupported output format \"%s\" - must be one of: %s", s, strings.Join(reporter.Format(), ", "))
 				},
 			},
 			&cli.BoolFlag{
 				Name:  "json",
 				Usage: "sets output to json (deprecated, use --format json instead)",
 			},
+			&cli.StringFlag{
+				Name:      "output",
+				Usage:     "save result output to file",
```

Comment:
nit: to be more clear, something like "save the result to the given file path" ?